tests/mbed/enc/CMakeLists.txt

```diff
@@ -22,9 +22,23 @@ function(add_mbed_test_enclave NAME)
 
   set(MBEDTLS_TESTS_DIR "${PROJECT_SOURCE_DIR}/3rdparty/mbedtls/mbedtls/tests")
 
+  if (WIN32)
+     find_program(GIT git)
+     get_filename_component(GIT_DIR ${GIT} DIRECTORY)
+     find_program(PERL perl
+        PATHS "C:/Program Files/Git/usr/bin" "${GIT_DIR}/../usr/bin"
+        NO_DEFAULT_PATH)
+  else()
+     find_package(Perl)
```

Comment:
On non-Windows we can (and should) just use `find_program` here since the Perl CMake package doesn't give us anything we're using (all it does is figure out the Perl version which we don't care about).

Reply:
I didn't want to change code for Linux unless absolutely necessary. This is how the mbedtls CMakeLists.txt is written (not sure if all those files are used but they are in our repo).

Reply:
I didn't want to change code for Linux unless absolutely necessary. This is how the mbedtls CMakeLists.txt is written (not sure if all those files are used but they are in our repo). But will incorporate.

Reply:
Let's keep the logic as consistent as possible. The Perl CMake package is unnecessary.

Reply:
Yep, is being removed.